src/Microsoft.OpenApi/Services/OpenApiVisitorBase.cs

```diff
@@ -12,6 +14,37 @@ namespace Microsoft.OpenApi.Services
     /// </summary>
     public abstract class OpenApiVisitorBase
     {
+        private readonly Stack<string> _path = new Stack<string>();
+       
+        /// <summary>
+        /// Allow Rule to indicate validation error occured a deeper context level.  
```

Comment:
>  [](start = 59, length = 1)

preposition missing

Reply:
still missing :)

---
In reply to: [162469699](https://github.com/Microsoft/OpenAPI.NET/pull/176#discussion_r162469699) [](ancestors = 162469699)

Reply:
Now it's fixed :-)